contracts/PetPark.sol

```diff
@@ -3,5 +3,80 @@ pragma solidity ^0.8.0;
 
 
 contract PetPark {
+  
+  struct Person {
+    uint gender;
+    uint age;
+    uint borrowedAnimal;
+  }
+  
+  address owner;
 
-}
\ No newline at end of file
+  mapping (uint => uint) animalToCount;
+  mapping (address => Person) addressToPerson;
+
+  event Added(uint animalType, uint animalCount);
+  event Borrowed(uint animalType);
+  event Returned(uint animalType);
+  
+  constructor() {
+    owner = msg.sender;
+  }
+
+  modifier onlyOwner {
+    require(msg.sender == owner, "Not an owner");
+    _;
+  }
+
+  function animalCounts(uint _animalType) view public returns (uint) {
```

Comment:
Since solidity public mappings are automatically exposed as functions, you can just make `animalCounts` as a public mapping